components/bio-formats-plugins/src/loci/plugins/macro/LociFunctions.java

```diff
@@ -725,6 +725,8 @@ public void run(String arg) {
       IJ.log("-- dimension order.");
       IJ.log("Ext.getMetadataValue(field, value)");
       IJ.log("-- Obtains the specified metadata field's value.");
+      IJ.log("Ext.getSeriesMetadataValue(field, value)");
```

Comment:
So is the value argument some kind of reference for where to put the gotten value?

Reply:
Yeah, essentially `value` will store the returned value after the call, so a very simple example would be:

`run("Bio-Formats Macro Extensions");
id = File.openDialog("Choose a file");
Ext.setId(id);
Ext.setSeries(0);
Ext.getSeriesMetadataValue("myMetadataField", value);
print("Returned value = " + value);`